Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -0,0 +1,311 @@
+| CV32E20 RTL + CORE-V-VERIF Integration		| Completed				|				|				|				|
+| Tool-chain and Programming Environment		| Completed				|				|				|				|
+| ImperasDV Reference Model Integration			| Completed				|				|				|				|
+| Spike Tandem Reference Model Integration		| Completed				|				|				|				|
+| Spike Model running with initial directed tests		| Completed				|				|				|				|
+| Random Instruction Generator (riscv-dv) Integration and Updates | 			|	Completed				|				|				|
+| OBIv1.2 Compliance								| 						|	Completed	|				|				|
+| Interrupt Verification						| 						|				| Completed		|				|
+| CSR Verification								| 						|	Completed	|				|				|
+| Counters Verification							| 						|				|				|	Completed	|
+| Exception Verification						| 						|				| Completed		|				|
+| Debug Verification							| 						|				| Completed		|				|
+| RISCOF										| Completed 			|				| 				|				|
+| Coverage Sign-off								| 						|				|				|	Completed	|
+| RTL Freeze 									| 						|				|				|	Completed	|
+| Release 1.0 									| 						|				|				|	Completed	|
+
+
+
+The schedule will be maintained with the following approach
+- The Project Board already established https://github.com/orgs/openhwgroup/projects/5/views/1will be updated to include the high level milestones here as well as detailed task list created in the planning spreadsheet
+- The Project Board will be regularly maintained by the Project Manager and will be used as a basis for discussion at weekly project meetings.
+
+
+
+## Project Repo 
+
+
+
+*The project repo(s) should be setup and listed here*
+*The repos should have README.MD and LICENSE.MD created*
+*The LICENSE.MD file should specify the project license*
+
+The project repo is https://github.com/openhwgroup/cve2
+The readme.md file is available https://github.com/openhwgroup/cve2/blob/main/README.md
+
+The license.md file is available but should be updated to indicate Solderpad 2.1
+https://github.com/openhwgroup/cve2/blob/main/LICENSE
+
+
+## Regular Project Meeting 
+*The project should have a regular project meeting established and included on the OpenHW calendar. Send a recurring meeting request to meetings@openhwgroup.org*
+Proposed regular project meeting every week at Thursday 8:00 Pacific / 11:00 Eastern Time/ 17:00 Central Europe 
+
+## Regular Project Report
+*The project should have a regular project report agreed. The format of the report should follow the suggestion in the programs/template repo. Consult OpenHW staff as meeting report location*.
+
+
+The monthly project report will be prepared for the OpenHW Cores TG meeting. 
+The report template at https://github.com/openhwgroup/programs/blob/master/process/Project-Monthly-Report-Template.md will be used.
+
+
+## Risk Register
+*Risks towards completion of the project for the given target should be listed and risk mitigation plan should be stated*
+
+
+| Risk								| Description						| Plan to Mitigate Risk					|      
```

Comment:
Lack of a Design Lead for documentation and RTL support.

Reply:
Ok, Let's discuss this one at the E20 project meeting.